openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -90,14 +90,19 @@
     ContentLibraryPermission,
     ContentLibraryBlockImportTask,
 )
-from openedx.core.djangoapps.content_libraries.signals import (
+
+
+from openedx_events.content_authoring.data import ContentLibraryData, LibraryBlockData
+
+from openedx_events.content_authoring.signals import (
```

Comment:
isort nit: should move these imports up underneath the `opaque_keys` imports above (and remove the blank lines between them). This is an external package, and so that's where they go.

Reply:
Fix: eecbbfa8d9fb7991afb993b51dce992570215336